starkup.sh

```diff
@@ -15,6 +15,19 @@ ASDF_SHIMS_ESCAPED="\${ASDF_DATA_DIR:-\$HOME/.asdf}/shims"
 LOCAL_BIN="${HOME}/.local/bin"
 LOCAL_BIN_ESCAPED="\${HOME}/.local/bin"
 
+BOLD=""
+RED=""
+YELLOW=""
+RESET=""
+
+# Check whether colors are supported and should be enabled
+if [ -z "${NO_COLOR:-}" ] && { [ -n "${TERM:-}" ] && echo "$TERM" | grep -q "^xterm"; }; then
```

Comment:
```suggestion
if [ -z "${NO_COLOR:-}" ] && grep -q "^xterm" <<< "$TERM"; then
```

should probably be enough

Reply:
Discussed offline, `<<<` seems to be unsupported in posix, so removed `[ -n "${TERM:-}" ]` to simplify check instead 👍🏻